addons/rating/models/rating.py

```diff
@@ -42,7 +42,7 @@ def new_access_token(self):
     parent_res_id = fields.Integer('Parent Document', index=True)
     rated_partner_id = fields.Many2one('res.partner', string="Rated person", help="Owner of the rated resource")
     partner_id = fields.Many2one('res.partner', string='Customer', help="Author of the rating")
-    rating = fields.Float(string="Rating", group_operator="avg", default=0, help="Rating value: 0=Unhappy, 10=Happy")
+    rating = fields.Float(string="Rating Value", group_operator="avg", default=0, help="Rating value: 0=Unhappy, 10=Happy")
```

Comment:
Nope, this is actually the rating. But maybe you can rename the field `rating_text` into Rating Text?